beheer.md

```diff
@@ -121,26 +121,36 @@ Beheren publicatie op GEMMA Online | Ja | Zelf
 Bij de ontwikkeling van Open API standaarden nemen diverse partijen deel. Het is handig voor ontwikkelaars maar ook voor beheerders om daar een overzicht van te hebben. Daarom dient de project manager of scrum master bij de aanvang van een ontwikkelproject maar ook tijdens het project als de teamsamenstelling wijzigt de [lijst met Deelnemende partijen](https://github.com/VNG-Realisatie/api-beheer/blob/master/Deelnemende%20partijen.md) in te vullen. Ook de beheerder of beheerders van de standaard moeten na in beheername van een standaard of bij wijziging van de beheerder op deze lijst worden vermeldt.
 
 ## Beheerproces
-Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam.
-Voor die betreffende standaard wordt dat beheerteam daarna verantwoordelijk voor:
+Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam. Teneinde de API standaard breed gedragen en ondersteund te laten worden is een community van belang. Deze community wordt gevormd door gemeenten, leveranciers en beheer. 
+Vanuit en samen met die community wordt dat beheerteam daarna voor die betreffende standaard verantwoordelijk voor:
 1. beantwoorden van vragen;
 2. het oplossen van problemen;
 3. doorontwikkeling van de standaard.
 
 **Beantwoorden van vragen**
+Vragen mbt. de betreffende standaard kunnen beheer op verschillende manieren bereiken:
+* Mail/telefoon
+* Discussieforum _(NB doen we nog iets met een discussieforum of verlaten we dat pad?)_
+* Slack
+* Issue in Github/Gitlab
+* Pullrequest in Github/Gitlab
 
-Voor het beantwoorden van vragen m.b.t. de betreffende standaard zijn twee kanalen beschikbaar. Ten eerste het specifiek voor de standaard beschikbaar gestelde kanaal in de Slack workspace [VNG API Community](https://vngapicommunity.slack.com) en ten tweede het eveneens specifiek voor de standaard beschikbaar gestelde GitHub/GitLab omgeving.
+Vragen via mail of telefoon hebben niet de voorkeur maar dit gaat ongetwijfeld voorkomen. Bij kleinere vragen hoeft dit geen probleem te zijn maar omwille van de openheid en de herbruikbaarheid van antwoorden kan het nuttig zijn deze alsnog via een discussieforum, Github/Gitlab of Slack te delen. Een andere optie om de vragen en antwoorden langdurig te delen is een veelgestelde vragenlijst samenstellen die in de loop van de tijd groeit. Zo'n lijst moet ergens goed vindbaar gepubliceerd worden, bijvoorbeeld op GEMMA Online of in de repository van de betreffende standaard. 
+
+Voor het beantwoorden van vragen m.b.t. de betreffende standaard zijn meerdere kanalen beschikbaar:
+1. Het specifiek voor de standaard beschikbaar gestelde kanaal in de Slack workspace [VNG API Community](https://vngapicommunity.slack.com) 
+2. Een eveneens specifiek voor (deze versie van) de standaard beschikbaar gestelde GitHub/GitLab omgeving.
+3. Een specifiek discussieforum voor de betreffende standaard.
```

Comment:
Niet dus. We moeten het aantal kanalen zo beperkt mogelijk houden.

Reply:
Discussieforum verwijderd